cypress/support/helpers.ts

```diff
@@ -0,0 +1,4 @@
+export const waitForPageToLoad = () => {
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(1000)
```

Comment:
This will always add a second to the test run whenever we call it. Could we use the following instead?
```
cy.get("h1", {timeout: 1000})
```

Reply:
We had this before and the tests were failing. We've had a look with @monotypical .

Reply:
Yeah, unfortunately with just waiting for h1 we would still sometimes get accessability violations with the tests being run before the page had fully loaded